app/assets/javascripts/actions/tickets_actions.js

```diff
@@ -149,6 +150,11 @@ export const deleteTicket = id => async (dispatch) => {
   dispatch({ type: DELETE_TICKET, id });
 };
 
+export const deleteNote = id => async (dispatch) => {
+  request(`/td/tickets/replies/${id}`, { method: 'DELETE' });
+  dispatch({ type: MESSAGE_KIND_NOTE_DELETE, id });
```

Comment:
It looks like this makes the request and then dispatches the action to remove the note from the frontend, which means that if the request fails, there's no error message and it still behaves as if the message got deleted.

See some of the other actions for typical patterns for handling this sort of request; the redux action should be based on the server response.

Reply:
yep, I will fix that case as well